e2e/runner/run_cypress_local.ts

```diff
@@ -106,11 +112,28 @@ const init = async () => {
     );
   }
 
+  switch (options.TEST_SUITE) {
```

Comment:
We can pass a test suite name to the function, and also define configs by a test suite name